src/main.rs

```diff
@@ -27,54 +30,83 @@ use crate::{request::RequestedPolicy, subscribe::read_last_timestamp};
 
 const REQUESTED_POLICY_CHANNEL_SIZE: usize = 1;
 const DEFAULT_TOML: &str = "/usr/local/aice/conf/crusher.toml";
-const USAGE: &str = "\
-USAGE:
-    crusher [CONFIG]
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+#[derive(Debug, Clone)]
+pub struct ManagerServer {
+    name: String,
+    rpc_srv_addr: SocketAddr,
+}
+
+impl FromStr for ManagerServer {
+    type Err = anyhow::Error;
+
+    fn from_str(manager_server: &str) -> Result<Self> {
+        let (name, rpc_srv_addr) = manager_server
+            .split_once('@')
+            .context("cannot get information of the manager server")?;
```

Comment:
```suggestion
            .context("cannot get information of the Manager server")?;
```

Reply:
Applied suggested adjustment.
I should have checked it with your previous comment. Thank you.